harness/setup.py

```diff
@@ -25,6 +25,7 @@
         "pyzmq>=18.1.0",
         "yogadl==0.1.4",
         # Common:
+        "backoff",
```

Comment:
why this?

Reply:
`backoff` was added recently and not sorted correctly according to the comments in the file.